node-module-system/calculate.js

```diff
@@ -0,0 +1,23 @@
+import add from './add.js';
+
```

Comment:
It's much more common to _not_ have a blank line between import statements, but to just have one blank line after all of them.